docs/reproducing_leaderboards.md

```diff
@@ -0,0 +1,70 @@
+# Reproducing Leaderboards
+
+You can use the HELM package to rerun evaluation runs and recreate a specific public leaderboard.
+
+The general procedure is to first find the appropriate `run_entries_*.conf` and `schema_*.yaml` files from the HELM GitHub repository for the leaderboard version, and then place them in your current working directory. The locations of these files are as follows:
```

Comment:
nit: do not need the comma after `for the leaderboard version`

Reply:
Fixed.